checkers_bot_tournament/game.py

```diff
@@ -93,6 +110,14 @@ def import_pdn(self, filename: str) -> None:
             self.move_number += 1
             self.swap_turn()
 
+    def move_piece(self, move: Move) -> Tuple[bool, bool]:
```

Comment:
There's stuff common between the PDN making piece moves and the run loop with the bots that you could combine. For example I think if `self.verbose` mode is on you also wanna print out the move and board display. You might wanna also detect for the game ending during the PDN (assert this doesn't happen cos why would you wanna). 

This is tangential but finally, I think you could make self.move_number an @property of len(self.move_history() + 1) so you don't have to update it anymore. Obviously then take caution about where to append the next move to move_history.

Reply:
> There's stuff common between the PDN making piece moves and the run loop with the bots that you could combine.

Yup sure

>  if self.verbose mode is on you also wanna print out the move and board display

verbose writes the games as a txt file, if it were to output to stdout it'll be huge

> self.move_number a property

Yeah sounds good 

>  take caution about where to append the next move to move_history.

This should be fine, I just add moves to the end of the move_history list

Reply:
> verbose writes the games as a txt file, if it were to output to stdout it'll be huge
I meant to the txt file not stdout...

> This should be fine, I just add moves to the end of the move_history list
I meant where to execute the appendment: at the start, end or somewhere in the middle -- because right now the appendment and increment of move counter are different places so you'll wanna be careful to not change any behaviour

Reply:
Moves are being made, just like in the run() loop, so I think it should get included in the txt file. It's just that they're mandated by the PDN so one of the lines could clarify, `Moved from x to x (Book Move)`

Reply:
> so one of the lines could clarify, Moved from x to x (Book Move)

Okok

Reply:
> I meant where to execute the appendment: at the start, end or somewhere in the middle

Yep I did fix this :D